Source/ProgressionSystemRuntime/ProgressionSystemRuntime.Build.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Valeriy Rotermel and Yevhenii Selivanov
+
+using UnrealBuildTool;
+
+public class ProgressionSystemRuntime : ModuleRules
+{
+	public ProgressionSystemRuntime(ReadOnlyTargetRules Target) : base(Target)
+	{
+		PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
+		CppStandard = CppStandardVersion.Latest;
+		bEnableNonInlinedGenCppWarnings = true;
+
+		PublicDependencyModuleNames.AddRange(new[]
+			{
+				"Core"
+				, "Bomber" // Is included in header files
```

Comment:
Please format it as next for better visibility:
```
		PublicDependencyModuleNames.AddRange(new[]
			{
				"Core"
				, "UMG"
				// Bomber modules
				, "Bomber" // Includes types in headers
			}
		);

		PrivateDependencyModuleNames.AddRange(new[]
			{
				"CoreUObject", "Engine", "Slate", "SlateCore" // Core
				// Bomber modules
				"MyUtils"
			}
		);
```

Reply:
improved